docs/SensitiveRecordDecrypt-transform.md

```diff
@@ -27,4 +27,8 @@ Metrics
 This plugin records three metrics:
 * `dlp.requests.count`: Total number of requests sent to Data Loss Prevention API
 * `dlp.requests.success`: Number of requests that were successfully processed by Data Loss Prevention API
-* `dlp.requests.fail`: Number of requests that failed
\ No newline at end of file
+* `dlp.requests.fail`: Number of requests that failed
+
+Custom Template Path
+-----------
+The availability to use a custom template path which is located in a different project other than the one specified in Project Id.
```

Comment:
nit: availability -> option. 
Same thing in SensitiveRecordFilter-splittertransform.md